geo_heatmap.py

```diff
@@ -205,7 +230,8 @@ def run(self, data_files, output_file, date_range, stream_data, tiles):
         "files", metavar="file", type=str, nargs='+', help="Any of the following files:\n"
         "1. Your location history JSON file from Google Takeout\n"
         "2. Your location history KML file from Google Takeout\n"
-        "3. The takeout-*.zip raw download from Google Takeout \nthat contains either of the above files")
+        "3. A GPX file containing GPS tracks\n"
+        "4. The takeout-*.zip raw download from Google Takeout \nthat contains either of the above files")

```

Comment:
Why is this list numbered? The numbers are irrelevant.

Reply:
Right. I changed that and also updated the readme with the new format support